### PR TITLE
P0 login signup

### PR DESCRIPTION
**Pull request by gauravsinhaweb**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -439,6 +439,123 @@ <h3>Xbox One Series S</h3>
...
+      <!-- login  -->
+      <div id="modal" class="modal">
+        <div class="card modal_box flex">
+          <div class="modal_img">
+            <div class="login_banner flex-center">
+              <img src="/assets/download.png" alt="login-img" />
+              <div class="nav_brand_text h3">
+                Pixel.<span class="store">Store</span>
+              </div>
+            </div>
```
Comment (pankaj-prog) [inline index.html]: Use semantics HTML

File: index.html [hunk 2]
```diff
@@ -439,6 +439,123 @@ <h3>Xbox One Series S</h3>
...
+          </div>
+          <div class="modal_content">
+            <div class="flex wrapper_head">
+              <div class="h2">Login</div>
```
Comment (pankaj-prog) [inline index.html]: Use header tag for headings

**Review by pankaj-prog — APPROVED** [review]
- Add to cart button in cards is an important element, but it looks inattentive. Make it attention grabbing.
- Gap between trending offers and a banner below it can be increased. 

Other than that the UI is clean.

File: index.html (hunk 1 above)
Comment (gauravsinhaweb) [reply index.html]: Made the changes, Thanks for pointing it out!

File: index.html (hunk 2 above)
Comment (gauravsinhaweb) [reply index.html]: Made the changes, Thanks for pointing it out!

**Review by manish-GH — APPROVED** [review]
Nice choice of theme and the UI looks clean. It can be merged

**Review by JuvitaSaini09 — APPROVED** [review]

**Pull request merged**